src/tools/rustdoc-gui/tester.js

```diff
@@ -172,12 +172,7 @@ async function main(argv) {
     files.sort();
 
     console.log(`Running ${files.length} rustdoc-gui tests...`);
-    if (opts["jobs"] < 1) {
-        process.setMaxListeners(files.length + 1);
-    } else {
-        process.setMaxListeners(opts["jobs"]);
-    }
-    let tests = [];
+    const tests = [];
```

Comment:
Maybe we should rename this symbol `running_tests` or `tests_queue` something alike

Reply:
Good idea.